itests/fe/permission_requests_test.py

```diff
@@ -1,187 +1,178 @@
-    expected_perms = [
-        "{}, {}".format(PERM_WITH_GRANTER, ARGUMENT),
-        "{}, {}".format(PERM_NO_GRANTER, ARGUMENT),
-        "{}, {}".format(PERMISSION_ADMIN, ARGUMENT),
-    ]
-    request_perms = [row.requested for row in request_rows]
-    assert sorted(expected_perms) == sorted(request_perms)
-
-    # Check the status change rows as well.  The timestamp part of the row is normally "(now)" but
-    # can be "(1 second ago)" on slow systems and "(in the future)" if MySQL rounds the timestamp
-    # up.  Allow for all of these.
-    expected_whos = ([REQUESTING_USER] * 3) + ([GRANTING_USER] * 2)
-    request_whos = []
-    for row in page.status_change_rows:
-        who, when = row.who.split(None, 1)
-        assert when in ("(now)", "(1 second ago)", "(in the future)")
-        request_whos.append(who)
-    assert sorted(expected_whos) == sorted(request_whos)
-
-    # and make sure the "no requests" row doesn't show up
-    with pytest.raises(Exception):
-        page.no_requests_row
-
-
-def test_outbound_requests(async_server, browser, do_request_perms):  # noqa: F811
-    fe_url = url(async_server, "/permissions/requests?direction=Requested+by+me")
-    browser.get(fe_url)
-
-    # Check that the request rows have info we expect, namely the 2 requests
-    # made by REQUESTING_USER but not the one request made by another user
-    page = PermissionRequestsPage(browser)
-    request_rows = page.request_rows
-    expected_perms = [
-        "{}, {}".format(PERM_WITH_GRANTER, ARGUMENT),
-        "{}, {}".format(PERM_NO_GRANTER, ARGUMENT),
-    ]
-    request_perms = [row.requested for row in request_rows]
-    assert sorted(expected_perms) == sorted(request_perms)
-
-    # Check the status change rows as well
-    sc_rows = page.status_change_rows
-    expected_groups = [REQUESTING_TEAM, REQUESTING_TEAM]
-    request_groups = [row.group for row in sc_rows]
-    assert sorted(expected_groups) == sorted(request_groups)
-
-    # and make sure the "no requests" row doesn't show up
-    with pytest.raises(Exception):
-        page.no_requests_row
-
-
-def test_no_requests(async_server, browser, do_action_requests):  # noqa: F811
-    fe_url = url(async_server, "/permissions/requests?status=pending&direction=Requested+by+me")
-    browser.get(fe_url)
-
-    page = PermissionRequestsPage(browser)
-    assert page.no_requests_row is not None
-
-    assert len(page.request_rows) == 0
-    assert len(page.status_change_rows) == 0
+
+if TYPE_CHECKING:
+    from py.path import LocalPath
+    from selenium.webdriver import Chrome
+    from tests.setup import SetupTest
+
+
+def create_permission_requests(setup: SetupTest) -> None:
+    """Create a permission requesting scenario.
+
+    Set up a permission requesting scenario in which REQUESTING_USER has both inbound and outbound
+    requests that they should be able to see on the requests page. Do this because the itest
+    infrastructure visits all pages as cbguder.
```

Comment:
Nits: `REQUESTING_USER` not a thing anymore, and no longer true that itests can only be as `cbguder`

Reply:
Thanks, fixed.